VowelsAndDuplicates.js

```diff
@@ -1,4 +1,16 @@
 function duplicatesAndVowels(name) {
-  
+  //sorting out vowels in a string
+  const vowels = name.match(/[aeiouAEIOU]/gi);

```

Comment:
Good work on the regex but since you are making it case insensitive by passing the `i` flag, you don't need `AEIOU` in the character set.

Reply:
okay. i thought it was case sensitive